nasa/earthdata-search
# EDSC-3284: Allows exporting granule searches as stac json

# Overview

### What is the feature?

 Allows exporting granule searches as stac json

### What is the Solution?

Allow users to download their current granules search as a stac json file

### What areas of the application does this impact?

Granules results page

# Testing

### Reproduction steps

- **Environment for testing:** SIT
- **Collection to test with:** ANY

1. Visit any collection
2. Download the stac json
3. Confirm the stac file matches search
4. View a collection with more than 1m granules
5. Confirm export button is disabled and tooltip displays on hover

# Checklist
